examples/src/main/java/alluxio/cli/JournalCrashTest.java

```diff
@@ -234,7 +233,7 @@ private static void killMaster() {
    *
    * Usage:
    * {@code java -cp
-   * alluxio-<ALLUXIO-VERSION>-jar-with-dependencies.jar alluxio.examples.JournalCrashTest}
+   * alluxio-<ALLUXIO-VERSION>-jar-with-dependencies.jar alluxio.cli.JournalCrashTest}
```

Comment:
instead of adding a comment here, print a usage message in the main function when it fails (similar to how AlluxioMaster or AlluxioWorker do this)